### ISSUE TITLE
Creating and saving/updating classes is very slow

### ISSUE THREAD

**Issue by tnolet** [joeferraro/MavensMate#676]
Creating a new class or simply saving an update to an existing class sometimes takes up to 1 minute. This makes using Mavens Mate almost impossible and really kills the workflow. I'm not sure how to resolve or debug this.

**Comment by tnolet** [comment]
related https://github.com/joeferraro/MavensMate/issues/631

**Comment by LKozakewycz** [comment]
I wish my saves would take 1 minute... you are a very lucky man.

**Comment by tnolet** [comment]
@LKozakewycz upgrading to SublimeText plugin V7 Beta and using the new background app fixed a lot.

**Comment by DarrenGood** [comment]
Same problem. I moved over from Eclipse because I was having so much trouble with. MM has been great for about a month and last couple of days some compiles take 5 minutes.... Utilities > Reset Metadata container didn't help. When I first started, compiling wouldn't work so I went to "mm_compile_with_tooling_api" : false
v7 doesn't look like it's ready for Windows (and I'm not sure how to install the MM-app). Any clues? Many thanks.

**Comment by LKozakewycz** [comment]
Hi guys - I went home. Solved all my issues... :laughing: 

In all seriousness, the above was correct. I simply went home, compiled my files and they now take a few seconds to compile. For me, it seems it was a network related issue. Which was strange because my colleagues were not experiencing the same issues and I was able to perform other web related stuff just fine.

This is not a coincidence - my files were taking taking from 10 minutes up to a couple of hours to save (for just a single file) for 6 days straight. I compiled one day after work when I got home, and everything worked great. I have now stayed at home for the entire day and have experienced no issues whatsoever.

That is not a solution - but that is exactly what I am doing right now.
